src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateGenerator.cs

```diff
@@ -0,0 +1,574 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Just update the pull comment: The new code is in program.cs. All other *.cs copied from existing infrastructure implementation.